src/resolvers/__tests__/collection.test.js

```diff
@@ -708,108 +708,108 @@ describe('Collection', () => {
             collection: null
           })
         })
+      })
 
-        describe('when retrieving revisions that contain tombstones', () => {
-          test('includes them in the results', async () => {
-            nock(/example-cmr/)
-              .defaultReplyHeaders({
-                'CMR-Hits': 1,
-                'CMR-Took': 7,
-                'CMR-Request-Id': 'abcd-1234-efgh-5678'
-              })
-              .get(/collections\.json/)
-              .reply(200, {
-                feed: {
-                  entry: [{
-                    id: 'C100001-EDSC'
-                  }],
-                  facets: {}
-                }
-              })
+      describe('when retrieving revisions that contain tombstones', () => {
```

Comment:
why remove this test?

Reply:
It wasn't removed I just noticed it was in the wrong place so I fixed it while I was here.